common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -84,74 +69,129 @@ const Tree = styled.div`
-      height: 0;
-    }
+  &.guideline::before {
+    border-left: 1px solid ${props => props.theme.color('yellow')};
+    height: 100%;
+    width: 0;
+    top: ${`${verticalGuidePosition}px`};
+    left: ${`${controlWidth / 2}px`};
+    height: calc(100% - ${`${verticalGuidePosition + controlHeight / 2}px`});
+  }
 
-    li::after {
-      border-left: 2px solid ${props => props.theme.color('teal')};
-      height: 100%;
-      width: 0px;
-      top: 10px;
-    }
+  &.guideline::after {
+    display: block;
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background: ${props => props.theme.color('yellow')};
+    left: ${`${controlWidth / 2 - 3}px`};
+    bottom: ${`${controlHeight / 2}px`};
+  }
+`;
 
-    li:last-child::after {
-      height: 10px;
-    }
+const TreeControl = styled.span`
+  display: inline-block;
+  cursor: pointer;
+  height: ${`${controlHeight}px`};
+  width: ${`${controlWidth}px`};
+  min-width: ${`${controlWidth}px`};
+  position: relative;
+  z-index: 1;
+  &::before {
+    content: '';
+    position: absolute;
+    height: ${`${circleHeight}px`};
+    width: ${`${circleWidth}px`};
+    // centre the circle in the control
+    top: ${`${(controlHeight - circleHeight) / 2}px`};
+    left: ${`${(controlWidth - circleWidth) / 2}px`};
+    background: ${props =>
+      props.highlightCondition === 'primary'
+        ? props.theme.color('yellow')
+        : props.highlightCondition === 'secondary'
+        ? props.theme.color('yellow', 'light')
+        : props.theme.color('smoke')};
+    border: ${props =>
+      props.highlightCondition === 'secondary'
+        ? `1px solid ${props.theme.color('yellow')}`
+        : `2px solid ${props.theme.color('white')}`};
+    border-radius: 50%;
+  }
+  .icon {
+    position: absolute;
+    z-index: 1;
+    // centre the icon in the control
+    top: ${`${(controlHeight - 24) / 2}px`}; // icons have a height of 24px
+    left: ${`${(controlWidth - 24) / 2}px`}; // icons have a width of 24px
   }
 `;
 
-const TreeItem = styled.li`
-  padding: 10px 10px 10px 0;
+const StyledLink = styled.a`
+  display: inline-block;
+  min-height: ${`${controlHeight}px`};
+  line-height: 1;
+  color: ${props => props.theme.color('viewerBlack')};
+  background: ${props =>
+    props.theme.color(props.isCurrent ? 'yellow' : 'transparent')};
+  cursor: pointer;
+  margin-left: ${props =>
+    props.hasControl ? `-${controlWidth / 2}px` : `${controlWidth / 2}px`};
+  padding-top: ${props => `${props.theme.spacingUnit}px`};
+  padding-left: ${props =>
+    props.hasControl
+      ? `${circleWidth / 2 + props.theme.spacingUnit}px`
+      : props.isCurrent
+      ? `${props.theme.spacingUnit}px`
+      : 0};
+  padding-right: ${props => `${props.theme.spacingUnit * 2}px`};
+  text-decoration: none;
   &:focus {
-    outline: ${props =>
-      !props.hideFocus ? `2px solid ${props.theme.color('black')}` : 'none'};
+    outline: ${props => (!props.hideFocus ? 'auto' : 'none')};
```

Comment:
Maybe `box-shadow: ${props => props.theme.focusBoxShadow}` for consistency?

https://github.com/wellcometrust/wellcomecollection.org/blob/432a8d3ec2bccbea7ad773a44e0f87afbeadd7c7/common/views/themes/default.js#L73